server/test/comment.spec.js

```diff
@@ -0,0 +1,126 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+chai.use(chaiHttp);
+
+let userToken;
+before(async () => {
+  const userResponse = await chai
+    .request(app)
+    .post('/api/v1/auth/login')
+    .send({
+      email: 'mofe@okoro.com',
+      password: 'soldier'
+    });
+  userToken = userResponse.body.data.token;
+});
+describe('User post comment route', () => {
```

Comment:
Good job handling all test cases possible.

Reply:
Thank you